src/App.tsx

```diff
@@ -8,11 +9,33 @@ const tabs: Tab[] = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+type Props = {};
+
+type State = {
+  selectedTab: string;
+};
+
+class App extends React.Component<Props, State> {
+  state: State = {
+    selectedTab: 'tab-1',
```

Comment:
I guess better put here link on the tab (for example tabs[0]) or rename it to selectedTabId